index.html

```diff
@@ -60,5 +60,156 @@ <h1 id="introduction" class="poppins-font">
+      <li>javascript</li>
+     </ul>
+     <button type="button" class="poppins-font button">See Project</button>
+    </section>
+    <!-- card 2 -->
+    <section class="container-bg-color project">
+      <img src="./images/img2.svg" alt="protfolio snapshot">
+      <h3 class="proj-title">Tonic</h3>
+      <ul class="proj-about poppins-font no-bullets">
+       <li class="proj-name ">CANOPY</li>
+       <li class="proj-type">
+         Back End Dev</li>
+       <li class="proj-year">
+         2022</li>
+      </ul>
+      <p class="proj-primarytxt poppins-font">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="no-bullets proj-list poppins-font">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+      </ul>
+      <button type="button" class="poppins-font button">See Project</button>
+     </section>
+    <!-- card 3 -->
+    <section class="container-bg-color project">
+      <img src="./images/img3.svg" alt="protfolio snapshot">
+      <h3 class="proj-title">Tonic</h3>
+      <ul class="proj-about poppins-font no-bullets">
+       <li class="proj-name ">CANOPY</li>
+       <li class="proj-type">
+         Back End Dev</li>
+       <li class="proj-year">
+         2022</li>
+      </ul>
+      <p class="proj-primarytxt poppins-font">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="no-bullets proj-list poppins-font">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+      </ul>
+      <button type="button" class="poppins-font button">See Project</button>
+     </section>
+    <!-- card 4 -->
+    <section class="container-bg-color project">
+      <img src="./images/img4.svg" alt="protfolio snapshot">
+      <h3 class="proj-title">Tonic</h3>
+      <ul class="proj-about poppins-font no-bullets">
+       <li class="proj-name ">CANOPY</li>
+       <li class="proj-type">
+         Back End Dev</li>
+       <li class="proj-year">
+         2022</li>
+      </ul>
+      <p class="proj-primarytxt poppins-font">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="no-bullets proj-list poppins-font">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+      </ul>
+      <button type="button" class="poppins-font button">See Project</button>
+     </section>
+  </div>
+  <!-- Footer -->
+  <section id="about-myself">
+    <!-- About myself -->
+    <div id="myself-1stchild">
+      <h2 class="main-heading poppins-font">About Myself</h2>
+      <p id="myselftxt" class="poppins-font intropara">
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="beforeicon-txt poppins-font">
+        LET’S CONNECT
+      </div>
```

Comment:
- [OPTIONAL] I think it would be better to use a header tag hx (`<h2`, `<h3>`, ...) for this element instead of <div> tag 👌. because It's considered a subtitle.